schemas/action-schema.json

```diff
@@ -56,6 +56,27 @@
                   "type": "string",
                   "description": "Target name",
                   "readOnly": false
+                },
+                "classname": {
```

Comment:
This is a POO "class" name, right? What about using a more abstract term like "component_name", so it could be used for other use-cases?

Reply:
That does make sense, we'll need to make some changes but for the best i guess

Reply:
On the other hand if other teams are not going to use them (e.g. `android_resource_id` will only be used by Android) maybe we should not make them part of the JSON schema.